statistics-production/ees.qmd

```diff
@@ -1204,14 +1239,14 @@ Once you're happy with your amendment it just needs to be [approved](#sign-off)
 
 ---
 
-## Methodology formatting
+<!--> ## Methodology formatting
```

Comment:
Was this intentional? 
![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/d825dacc-bb10-4637-b2f7-a1630818e43d)

Reply:
@cjrace kind of.. was hoping to have hashed it out for now (sorry I should have checked it), as thought it would be good for us to add in a section on using headers in general (whether main content/methodology/data guidance etc)

Reply:
I'd want to leave this in (if nothing else so I can double check what level header works for it as methodologies create a special table of contents from headers at certain levels), maybe we just rename the section to methodology subheadings?

Reply:
yeah sure, will add that in now